components/director/internal/domain/scenarioassignment/repository.go

```diff
@@ -14,22 +14,41 @@ var tenantColumn = "tenant_id"
 
 func NewRepository(conv EntityConverter) *repository {
 	return &repository{
-		creator: repo.NewCreator(tableName, columns),
-		conv:    conv,
+		creator:      repo.NewCreator(tableName, columns),
+		singleGetter: repo.NewSingleGetter(tableName, tenantColumn, columns),
+		conv:         conv,
 	}
 }
 
 type repository struct {
-	creator repo.Creator
-	conv    EntityConverter
+	creator      repo.Creator
+	singleGetter repo.SingleGetter
+	conv         EntityConverter
 }
 
 //go:generate mockery -name=EntityConverter -output=automock -outpkg=automock -case=underscore
 type EntityConverter interface {
 	ToEntity(assignment model.AutomaticScenarioAssignment) Entity
+	FromEntity(assignment Entity) model.AutomaticScenarioAssignment
 }
 
 func (r *repository) Create(ctx context.Context, model model.AutomaticScenarioAssignment) error {
 	entity := r.conv.ToEntity(model)
 	return r.creator.Create(ctx, entity)
 }
+
+func (r *repository) GetByScenarioName(ctx context.Context, tnt, scenarioName string) (model.AutomaticScenarioAssignment, error) {
+	var ent Entity
+
+	conditions := repo.Conditions{
+		repo.NewEqualCondition("scenario", scenarioName),
```

Comment:
The string "scenario" appears in a few places here, please move it to a variable (eg. `scenarioColumn`) like `tenantColumn` is done